errors/errors.go

```diff
@@ -22,7 +22,6 @@ const (
 	RejectedIdentifier
 	InvalidEmail
 	ConnectionFailure
-	WrongAuthorizationState
```

Comment:
This is the one piece of this change I am unsure of. Should this replace `WrongAuthorizationState` with `_`, instead of deleting it entirely, to preserve the numbering of the errors below it in the enum?

Reply:
Do we ever depend on the numbering in our code?

Reply:
I'd have to double check, but it's good odds that we use it when serializing these across the RPC boundary. That would argue for doing `_`.

Reply:
Yeah, that's what I was worried about as well. I did it this way anyway because I was thinking that there would inevitably by cross-version incompatibilities during deployment _anyway_ (which is true), but really doing it this way leads to _many, very bad_ incompatibilities whereas replacing it with `_` leads to _few, just for this one error type_ incompatibilities.

I've changed it.